lib/AliasPlugin.js

```diff
@@ -30,6 +31,25 @@ module.exports = class AliasPlugin {
 	 */
 	apply(resolver) {
 		const target = resolver.ensureHook(this.target);
+		const cache = {};
```

Comment:
The cache looks like an overkill for me here... Does it really make sense here?

Reply:
hm.. since mostly we won't have absolute aliases on average algorithm complexity will be `O(1)`.. lets remove it.